docs/quickstarts/insights-integrate-inventory-aap/insights-integrate-inventory-aap.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: insights-integrate-inventory-aap
+  externalDocumentation: true
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+  displayName: Integrating Insights as an inventory source in Ansible Automation Platform
+  icon: ~
+  description: Configuring Insights as an inventory source in AAP to pull system data into automation workflows.
```

Comment:
Just a suggestion: You might want to shorten the description a little to eliminate repetition from the card title - if this makes sense, maybe "Configure Insights to pull system data into AAP automation workflows." or similar. Also the convention the cards use for the description is a verb like 'Configure', 'Connect' etc (not a gerund). See:
https://www.uxd-hub.com/entries/resource/best-practices-for-writing-quick-starts#best-practices-for-writing-quick-starts_heading_143 Otherwise looks perfect to me!